lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandler.java

```diff
@@ -157,7 +157,11 @@ private void processErrorAsyncCriResponse(ErrorAsyncCriResponse errorAsyncCriRes
 
         criResponseItem.ifPresent(
                 responseItem -> {
-                    responseItem.setStatus(CriResponseService.STATUS_ERROR);
+                    if ("access_denied".equals(errorAsyncCriResponse.getError())) {
+                        responseItem.setStatus(CriResponseService.STATUS_ABANDON);
+                    } else {
```

Comment:
It would be good to have a way of noticing any new response error codes.
Either by logging an error here, or even throwing an exception if the code isn't one we recognise.